docs/admin/installation.md

```diff
@@ -12,10 +12,56 @@ modality: "universal"
 
 # Installation Guide
 
-This guide covers installing NeMo Curator and verifying your installation is working correctly.
+This guide covers installing NeMo Curator with support for **all modalities** and verifying your installation is working correctly.
 
 ## Before You Start
 
+### System Requirements
+
+For comprehensive system requirements and production deployment specifications, see [Production Deployment Requirements](deployment/requirements.md).
+
+**Quick Start Requirements:**
+
+- **OS**: Ubuntu 24.04/22.04/20.04 (recommended)
+- **Python**: 3.10, 3.11, or 3.12
+- **Memory**: 16GB+ RAM for basic text processing
+- **GPU** (optional): NVIDIA GPU with 16GB+ VRAM for acceleration
+
+### Install FFmpeg and Encoders
+
+Curator’s video pipelines rely on `FFmpeg` for decoding and encoding. If you plan to encode clips (for example, using `--transcode-encoder libopenh264` or `h264_nvenc`), install `FFmpeg` with the corresponding encoders.
+
+::::{tab-set}
+
+:::{tab-item} Debian/Ubuntu (Script)
+
+Use the maintained script in the repository to build and install `FFmpeg` with `libopenh264` and NVIDIA NVENC support. The script enables `--enable-libopenh264`, `--enable-cuda-nvcc`, and `--enable-libnpp`.
+
+- Script source: [docker/common/install_ffmpeg.sh](https://github.com/NVIDIA-NeMo/Curator/blob/main/docker/common/install_ffmpeg.sh)
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/NVIDIA-NeMo/Curator/main/docker/common/install_ffmpeg.sh -o install_ffmpeg.sh
+chmod +x install_ffmpeg.sh
+sudo bash install_ffmpeg.sh
+```
+
+:::
+
+:::{tab-item} Verify Installation
+
+Confirm that `FFmpeg` is on your `PATH` and that at least one H.264 encoder is available:
+
+```bash
+ffmpeg -hide_banner -version | head -n 5
+ffmpeg -encoders | grep -E "h264_nvenc|libopenh264|libx264" | cat
+```
+
+If encoders are missing, reinstall `FFmpeg` with the required options or use the Debian/Ubuntu script above.
+
+:::
+
+::::
+
 ### InternVideo2 Support (Optional)
```

Comment:
InternVideo2 also has two ways for installation: either PyPI installation (what you have here), or Source installation , so maybe better to have two tabs as well:
For source installation, we just need the additional two 
```
bash external/intern_video2_installation.sh
uv add InternVideo/InternVideo2/multi_modality
```
So I would suggest to have this **after** Installation Methods